.github/workflows/ci.yaml

```diff
@@ -55,16 +55,21 @@ jobs:
         juju:
           - agent: 2.9.45
```

Comment:
1) 2.9.46 is the latest
2) Dragomir has a renovate configuration to update this automatically. Consider to add it for MySQL codebase. Tnx!